snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -3091,10 +3041,10 @@ static void AppendEx()
         {
             // <Snippet201>
             // Creating a list of numbers
-            List<int> numbers = new List<int> { 1, 2, 3, 4 };
+            List<int> numbers = [1, 2, 3, 4];
 
             // Trying to append any value of the same type
-            numbers.Append(5);
+            _ = numbers.Append(5);

```

Comment:
This is an interesting change, and could benefit from an explanation:

The original code "looked like" calling a `void` return method that modified the receiver. That's a reasonably common mistake. Assigning the result to a discard highlights the mistake. That minimizes the impact of the sample as a learning tool.